batch/src/main/scala/uk/gov/ons/addressindex/models/HybridAddressNisraEsDocument.scala

```diff
@@ -93,7 +96,7 @@ object HybridAddressNisraEsDocument extends EsDocument {
     "organisationName" -> splitAndCapitalise(Option(row.getString(5)).getOrElse("")),
     "departmentName" -> splitAndCapitalise(Option(row.getString(6)).getOrElse("")),
     "subBuildingName" -> splitAndCapitalise(Option(row.getString(7)).getOrElse("")),
-    "buildingName" -> splitAndCapitalise(Option(row.getString(8)).getOrElse("")),
+    "buildingName" -> (if ("\\d+[A-Z]".r.findFirstIn(Option(row.getString(8)).getOrElse("")).isEmpty) splitAndCapitalise(Option(row.getString(8)).getOrElse("")) else Option(row.getString(8)).getOrElse("")),
```

Comment:
Is there something special about NISRA building names?

Reply:
The REGEX is used by the non-NISRA documents also, to ensure that building names like "1A" keep the suffix in caps rather than become "1a". This convention is not universal, but the most commonly used.

Reply:
Ok I get it now.